media/system/webcomponents/js/joomla-toolbar-button-es5.js

```diff
@@ -1,136 +0,0 @@
-(function(){function r(e,n,t){function o(i,f){if(!n[i]){if(!e[i]){var c="function"==typeof require&&require;if(!f&&c)return c(i,!0);if(u)return u(i,!0);var a=new Error("Cannot find module '"+i+"'");throw a.code="MODULE_NOT_FOUND",a}var p=n[i]={exports:{}};e[i][0].call(p.exports,function(r){var n=e[i][1][r];return o(n||r)},p,p.exports,r,e,n,t)}return n[i].exports}for(var u="function"==typeof require&&require,i=0;i<t.length;i++)o(t[i]);return o}return r})()({1:[function(require,module,exports){
```

Comment:
This should be done in upstream or not?

Reply:
https://github.com/joomla/joomla-cms/pull/21328/files#diff-35fee772e3a0759daa23b54977e344a6
@laoneo As i can see this things are already done in upstream but. I dont know why its not chaged here. ?
What should i do ?